.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3.8'
+# create dev.env with the following values:
+# OC_URL
+# OC_USER
+# OC_PASS
+# OC_PLUGIN_NAME
+services:
+  console:
+    build:
+      context: ..
+      dockerfile: .devcontainer/Dockerfile.console
+    env_file: dev.env
+    restart: unless-stopped
+    healthcheck:
+      test: oc get projects
```

Comment:
nit: If there are a lot of projects on the cluster, this is actually somewhat expensive. Are we sure it works?

```
$ oc get projects
Unable to connect to the server: dial tcp: lookup api.jcaiani93.devcluster.openshift.com on 192.168.4.1:53: no such host
$ echo $?
0
```